create.go

```diff
@@ -116,6 +130,42 @@ func getConfigs(bundlePath, containerID, console string) (vc.PodConfig, ShimConf
 	return *podConfig, shimConfig, *ociSpec, nil
 }
 
+func createCgroupsFiles(cgroupsPath string, pid int) error {
+	if cgroupsPath == "" {
+		return fmt.Errorf("Missing cgroups path")
+	}
+
+	cgroupsDir := filepath.Join(cgroupsMemDirPath, cgroupsPath)
```

Comment:
... and looking at the spec on cgroups again more closely, we should check whether `cgroupsPath` is absolute or relative. If we don't care, we should still document that approach.